### DISCUSSION TITLE
v0.9.1-beta

### DISCUSSION THREAD

**Discussion by sheikhhaziq** [sheikhhaziq/gyawun_music#76] [Announcements]
## What's Changed
* Fix: Corrected loading of playlists on the home page. by @mrmonterrosa in https://github.com/sheikhhaziq/vibemusic/pull/71

## New Contributors
* @mrmonterrosa made their first contribution in https://github.com/sheikhhaziq/vibemusic/pull/71

**Full Changelog**: https://github.com/sheikhhaziq/vibemusic/compare/v0.9.0-beta...v0.9.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/sheikhhaziq/vibemusic/releases/tag/v0.9.1-beta'>v0.9.1-beta</a>.</em>

**Comment by senolfndk** [comment]
song download very slow

**Reply by owenconnorz** [reply]
I think this is being worked on not 100%

**Comment by rianaokel** [comment]
Did someone fix the download problem ? songs don't download when requesting download from player.
